.travis.yml

```diff
@@ -28,6 +28,8 @@ install:
 - npm install;
 
 script:
+- git ls-tree -r master --name-only | grep ".*.[t|j]s$" | xargs ./node_modules/prettier/bin/prettier.js --write --print-width 100 --single-quote --trailing-comma es5
```

Comment:
I was thinking that we should be checking all the files to see if any of them haven't been prettified.

I think I set up `gulp prettier` to only prettify files that have changed since the last commit.

Come to think of it, I think this workflow can be improved...